.dockerignore

```diff
@@ -3,7 +3,6 @@ deps/
 charts/
 .vscode/
 log/
-.git/
```

Comment:
why remove this from dockerignore?

Reply:
We need the `.git` directory when building the container since Elixir will get the version from the Git tag.